### ISSUE TITLE
SET with arguments

### ISSUE THREAD

**Issue by ghostganz** [yeahoffline/redis-mock#29]
It appears that `SET` with arguments isn't working, i.e it tries to use `'EX'` as the callback when calling `client.set(key, value, 'EX', 10)`.

This example is even used in the redis client docs, so you would expect it to work:
https://github.com/NodeRedis/node_redis#redis-commands

**Comment by yeahoffline** [comment]
https://github.com/yeahoffline/redis-mock/blob/master/test/redis-mock.strings.test.js#L67 

?

**Comment by ghostganz** [comment]
... yes?

**Comment by AlecTroemel** [comment]
@ghostganz I just tested this in node `v8.9.0`, linking my npm dependency straight to this repo `    "redis-mock": "https://github.com/yeahoffline/redis-mock/tarball/master"`, and its working for me.. not sure if that helps at all..

**Comment by murillo128** [comment]
It seems that the SET always expect the last argument to be a callback:

https://github.com/yeahoffline/redis-mock/blob/master/lib/redis-mock.js#L363

so this
```
this.cache.set( this.key, JSON.stringify(this.entry), "EX", timeout );
```

fails with `callback is not a function`

**Comment by kamiljano** [comment]
this should now be fixed starting version 0.55.0
